src/Stacktrace.php

```diff
@@ -129,6 +129,11 @@ public function addFrame(string $file, int $line, array $backtraceFrame): void
         }
 
         if (isset($backtraceFrame['class']) && isset($backtraceFrame['function'])) {
+            $anonymousClassPrefix = 'class@anonymous' . "\x00";
+            if (0 === strpos($backtraceFrame['class'], $anonymousClassPrefix)) {
+                $backtraceFrame['class'] = $anonymousClassPrefix . $this->stripPrefixFromFilePath(substr($backtraceFrame['class'], \strlen($anonymousClassPrefix)));
```

Comment:
I'm not sure why but php-cs-fixer insists on adding a backslash to the `strlen` call but leaves other PHP functions alone.

Reply:
It's normal, we're using the @Symfony:risky ruleset that defines that only the functions that are specially optimized by Zend will be prefixed with the backslash

Reply:
It's normal, we're using the `@Symfony:risky` ruleset that defines that only the functions that are specially optimized by Zend will be prefixed with the backslash